### DISCUSSION TITLE
Can't authenticate Copilot

### DISCUSSION THREAD

**Discussion by wadefleming-nz** [community/community#20282] [Copilot Conversations] [answered]
I previously had 2 github accounts - call them personal and work (this account: wadefleming-nz)

The personal account had Copilot when it was free. When the started the 60 day trial, I attempted to sign up and gave my payment details but it failed to activate properly.

I have now retired the personal account and want to use the work one only.

- Signing into that account in VS Code and attempting to enable Copilot gives 'User not authorized'. 
- Going to the billing page for that account shows 'Your free trial expires in 42 days' under Copilot.
- Going to the settings page shows the Copilot options as if it is activated.

How can I get Copilot to work with the wadefleming-nz account?

VS Code logs:
```
[INFO] [auth] [2022-07-08T00:49:37.655Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2022-07-08T00:49:37.658Z] GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"
```

**Comment by davecheney** [comment, marked as answer]
@wadefleming-nz I can see no requests from the account `wadefleming-nz` in the last 24 hours. 
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Reply by wadefleming-nz** [reply]
Thanks I restarted VS code and it showed me logged into `wadefleming-nz` but also had an option to sign into Github, did that and it is activated. It's not auto suggessting though but I'll try get that working and make another post if I need to.

**Reply by modaye** [reply]
GitHub copilot has been unable to work. Although my account has been logged in a long time ago, it only rotates without any prompts. Use the new idea, which prompts you to try login again
